src/entities.py

```diff
@@ -1,18 +1,28 @@
 # -*- coding: utf-8 -*-
 
+import datetime
 import json
 import re
-from typing import Dict, Optional, Sequence, Union, List, Any, Set
-from datetime import datetime
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Sequence
+from typing import Set
+from typing import Union
```

Comment:
Why not as a list?
Do you know which rule demands this?

Reply:
this is because I copied the [isort](https://pypi.org/project/isort/) config option of `force_single_line = true` from another dev..
the reason he included it is as far as I can tell, because this makes diffs a bit eazier to review, but you are the juge...

Reply:
too be fair I'm too not really a fan of this style..
I'll have a look in the evening